ldap/servers/slapd/accesslog.c

```diff
@@ -0,0 +1,495 @@
+/** BEGIN COPYRIGHT BLOCK
+ * Copyright (C) 2024 Red Hat, Inc.
+ * All rights reserved.
+ *
+ * License: GPL (version 3 or any later version).
+ * See LICENSE for details.
+ * END COPYRIGHT BLOCK **/
+
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include "slap.h"
+
+#define JBUFSIZE 75
```

Comment:
This is what we use for the audit log JSON format.  It's not used yet, but it will be and I think the size is ok for what it's used for.  See auditlog.c